v5_MIGRATION_GUIDE.md

```diff
@@ -212,3 +213,73 @@ class MyActivity : FragmentActivity() {
 }
 ```
 
+## Local Payment
+
+The Local Payment integration has been updated to allow for more flexibility with browser 
+switching and result handling.
+
+`LocalPaymentLauncher` has been added to handle the browser switching portion of the local payment 
+flow for payment authorization via bank login. This class is used to handle browser switch
+results upon returning to your app from the web flow, therefore must be instantiated in or
+before the `OnResume` method of your Activity or Fragment.
```

Comment:
nit: Can we remove the custom `\n` line breaks? That'll let markdown handle the line wrapping when users increase the font size in their browsers.

Reply:
I think this is fixed - it was a setting in my Android Studio causing auto-wrap